packages/mdx/mdx.ts

```diff
@@ -0,0 +1,72 @@
+import { promises as fsPromises } from "fs";
+import path from "path";
+
+type MdxPath = {
+  params: {
+    fileName: Array<String>
+  }
+}
+
+/**
+ * Get all MD/MDX files paths on a folder and his subfolder.
+ * Returns an array of {params: {fileName: [ 'exampleSubFolder', 'exampleFile' ]}
```

Comment:
Then please add this structure as the return type of the function, smth like `Array<Path<MdxFile>>` or smth like that.

Reply:
Btw it feels like this is a Next.js specific function. To be discussed later, at the moment you can focus on typing it correctly